fiftyone_pipeline_core/fiftyone_pipeline_core/flowdata.py

```diff
@@ -65,24 +64,25 @@ def process(self):
         """
 
         if not self.processed:
+            # filter evidence before processing
+            try:
+                self.evidence.filter()
```

Comment:
please check if there are already tests verifying that evidence filtering works.  if not - please add them.  everything else looks good

Reply:
It works, at least these tests are passing

https://github.com/51Degrees/pipeline-python/blob/9fcc7372238e521c77e7f1d7598a34439675498f/fiftyone_pipeline_core/tests/test_core.py#L47-L59